src/main/java/ch/njol/skript/variables/FlatFileStorage.java

```diff
@@ -97,6 +105,7 @@ protected boolean load_i(final SectionNode n) {
 		boolean update2_0_beta3 = false;
 		final Version v2_1 = new Version(2, 1);
 		boolean update2_1 = false;
+		final Version post_2_3 = new Version("INSERT VERSION");
```

Comment:
Variable name should be updated to the version this'll be introduced in

Reply:
*should* be good now